CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Changes
 
+  * [#761](https://github.com/toptal/chewy/pull/761): Avoid fetching scope data to check if it is blank ([@dalthon][])
```

Comment:
Let's follow the [contributing guide](https://github.com/toptal/chewy/blob/master/CONTRIBUTING.md):
```
If the change has a related GitHub issue (e.g. a bug fix for a reported issue), put a link to the issue as [#123](https://github.com/toptal/chewy/issues/123): 
```
* [#716](https://github.com/toptal/chewy/issues/716): Avoid fetching scope data to check if it is blank ([@dalthon][])